configurer/configurer.go

```diff
@@ -15,7 +15,7 @@ const (
 
 //go:generate counterfeiter -o fakes/fake_configurer.go . RouterConfigurer
 type RouterConfigurer interface {
-	Configure(routingTable models.RoutingTable) error
+	Configure(routingTable models.RoutingTable, forceHealthCheckFailure bool) error
```

Comment:
**Suggestion**

`forceHealthCheckToFail` is more descriptive than `forceHealthCheckFailure` to me, but I don't think it matters a whole lot either way.

Reply:
I agree. I like Andy's suggestion `forceHealthCheckToFail` better.